apps/dashboard/app/(app)/identities/[identityId]/page.tsx

```
@@ -54,115 +57,158 @@ export default async function Page(props: Props) {
+          ) : (
+            <Table>
+              <TableHeader>
+                <TableRow>
+                  <TableHead>Name</TableHead>
+                  <TableHead>Limit</TableHead>
+                  <TableHead>Duration</TableHead>
+                </TableRow>
+              </TableHeader>
+              <TableBody>
+                {identity.ratelimits.map((ratelimit) => (
+                  <TableRow>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)